ccd-definition/CaseEventToComplexTypes/DefendantBankAccounts.json

```diff
@@ -0,0 +1,26 @@
+[
```

Comment:
If you create a new file in LR journey, please use the suffix LRspec in filename.. to distinguish it from the ones used by Damages